admin/src/main/resources/io/buoyant/admin/js/src/router_clients.js

```diff
@@ -22,7 +26,9 @@ define([
       return numClients < EXPAND_CLIENT_THRESHOLD;
     }
 
-    return function (metricsCollector, routers, $clientEl, routerName, clientTemplate, metricPartial, clientContainerTemplate, colors) {
+    return function (metricsCollector, routers, $clientEl, routerName, colors) {
+      var clientContainerTemplate = Handlebars.compile(routerClientContainerTemplate);
```

Comment:
we only instantiate RouterClients once (in RouterController) - so I think we are safe!